packages/pockethost.io/src/components/Navbar.svelte

```diff
@@ -1,30 +1,8 @@
 <script lang="ts">
   import MediaQuery from '$components/MediaQuery.svelte'
   import ThemeToggle from '$components/ThemeToggle.svelte'
-  import { client } from '$src/pocketbase'
-  import { createCleanupManagerSync } from '$util/CleanupManager'
-  import { onDestroy, onMount } from 'svelte'
-  const { isLoggedIn, logOut, onAuthChange } = client
-
-  const cm = createCleanupManagerSync()
-
-  let _isLoggedIn = isLoggedIn()
-  onMount(() => {
-    _isLoggedIn = isLoggedIn()
-    const unsub = onAuthChange(() => {
```

Comment:
I think you should still listen for this event somewhere 👍

Reply:
are you referring to the `onAuthChange()` ?

Reply:
Yes. The logged in/out status could change via some backend action (timeout, deauthorization, etc) so I think that we should listen for this event and update the store accordingly. I'm not really clear on all the scenarios where the event fires though. I could be making this all up :)

Reply:
Added back in and moved to the `stores.ts` file.